metricbeat/etc/fields.yml

```diff
@@ -8,7 +8,28 @@ defaults:
   ignore_above: 1024
 
 sections:
+  - ["general", "General event data"]
   - ["redis-info", "Redis Info Stats"]
+
+general:
+  type: group
+  description: >
+    General fields for each event
+  fields:
+    - name: response_time
```

Comment:
I don't think using millis for response_time would make sense, as quite often the value would be 0. As far as I understood, field formatter the way we need it here is not supported yet, but could be added easily. 

@tsg For your last comment: So you would go with `responsetime` for consistency reason (we don't break any BC here)?

Other option: Going with `response_time` and change it to `response_time` in packetbeat in the next major version but having it in nano seconds too. A config flag would be used to enable `responsetime` as millis in packetbeat for the next major release and then removed. This would be my favorite option. I think also packetbeat could profit from using nanos.